tests/e2e/util.go

```
@@ -1070,6 +1083,30 @@ func scaleTKGWorker(wcpHost string, wcpToken string, tkgCluster string, tkgworke
 
 }
 
+//deleteTKG method deletes the TKG Cluster
+func deleteTKG(wcpHost string, wcpToken string, tkgCluster string) {
```

You need to return the error